recipes/mputil/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "mputil" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6e510e587cd812055cd1b61eb311894bf5ed1ec1f81ad9b2b3413ef4df8aeb08" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
please use ``pip`` to install the package, like
```
python -m pip install --no-deps --ignore-installed .
```
Please also add ``pip`` as a build requirement

Reply:
Please also add
``noarch: python``